### ISSUE TITLE
Sync Gateway RPM examples directory missing executable permission

### ISSUE THREAD

**Issue by tleyden** [couchbase/sync_gateway#1010]
I'm getting these errors when trying to extract the RPM:

```
 rpm2cpio couchbase-sync-gateway-enterprise_1.1.0-28_x86_64.rpm | cpio -idmv
./opt/couchbase-sync-gateway
./opt/couchbase-sync-gateway/LICENSE.txt
./opt/couchbase-sync-gateway/README.txt
./opt/couchbase-sync-gateway/VERSION.txt
./opt/couchbase-sync-gateway/bin
./opt/couchbase-sync-gateway/bin/sync_gateway
./opt/couchbase-sync-gateway/examples
cpio: ./opt/couchbase-sync-gateway/examples/README.md: Cannot open: Permission denied
./opt/couchbase-sync-gateway/examples/README.md
```

On closer inspection, I can see the `examples` directory is missing executable permission:

```
[centos@ip-10-109-190-236 couchbase-sync-gateway]$ pwd
/tmp/opt/couchbase-sync-gateway
[centos@ip-10-109-190-236 couchbase-sync-gateway]$ ls -alh
total 24K
drwxr-xr-x. 5 centos centos 4.0K Jul 28 20:23 .
drwxrwxr-x. 3 centos centos   35 Jul 28 20:23 ..
drwxr-xr-x. 2 centos centos   25 Jul 28 20:23 bin
drw-r--r--. 2 centos centos    6 Jun 17 17:35 examples
-rw-r--r--. 1 centos centos 2.5K Jun 17 17:35 LICENSE.txt
-rw-r--r--. 1 centos centos   44 Jun 17 17:35 manifest.txt
-rw-r--r--. 1 centos centos  260 Jun 17 17:35 manifest.xml
-rw-r--r--. 1 centos centos 1.4K Jun 17 17:35 README.txt
drwxr-xr-x. 3 centos centos   83 Jul 28 20:23 service
-rw-r--r--. 1 centos centos    9 Jun 17 17:35 VERSION.txt
```

Actual: examples has permissions **drw-r--r--.**
Expected: **drwxr-xr-x**

**Comment by zgramana** [comment]
@michaelkwok can you please update?

**Comment by andreibaranouski** [comment]
I see the same thing on couchbase-sync-gateway-enterprise_1.1.1-8_x86_64.rpm

**Comment by ajres** [comment]
The examples directory permissions are now set during post installation, this has been tested by deploying the dinary distribution.

The fix has been merged onto a branch off master and PR #1249 has been raised.

Closing out this ticket.

**Comment by tleyden** [comment]
Great!  Thanks @ajres
